### DISCUSSION TITLE
How to do simple LEFT JOIN with QueryOver in NH 4.0?

### DISCUSSION THREAD

**Discussion by Magenta94** [nhibernate/nhibernate-core#2623] [Q&A]
I have NHibernate 4.0 and unfortunately I can't update right now, so instead of LINQ I have to use QueryOver, but I can't understand from the docs how to do it..

I have Table A and Table B and I want to do a left join on A.Id == B.Id

**Comment by gudmundurm** [comment]
Highly recommend reading through these docs by Andrew Whitaker.

http://www.andrewwhitaker.com/queryover-series/
